PelotonEppSdk/Models/EventResponse.cs

```diff
@@ -17,22 +17,24 @@ public class EventResponse : Response
         /// The name for the event.
         /// </summary>
         public string Name { get; set; }
+
+        public string FriendlyUrlPath { get; set; }
         /// <summary>
         /// The description for the event.
         /// </summary>
         public string Description { get; set; }
         /// <summary>
         /// The start datetime for the event.
         /// </summary>
-        public DateTime StartDatetime { get; set; }
```

Comment:
Ah, good catch. 
There is only minor reasoning for updating the names of the StartDatetime and EndDatetime fields to be StartDate and EndDate -- and that is to match the API names. 
But I would be comfortable with leave reverting the rename to avoid breaking backwards compatibility.
Thoughts on reverting the rename, @cattiwill, @nothinhappened ?
Note that the rename of the internal, snake-cased start_datetime and end_datetime to start_date and end_date is required to match the EPP API.